libpod/util.go

```diff
@@ -32,3 +34,11 @@ func StringInSlice(s string, sl []string) bool {
 	}
 	return false
 }
+
+// FuncTimer helps measure the execution time of a function
```

Comment:
Was this committed accidentally? I'm not opposed to keeping it, just wondering

Reply:
no accidently ... figured we might need something like this as we address these kind of issues.

Reply:
i mean NOT accidently.